.gitignore

```
@@ -1 +1,5 @@
-vendor
\ No newline at end of file
+vendor
+
+nodejs/welcome-email/node_modules
+nodejs/welcome-email/package-lock.json
```

This would become too specific to `welcome-email`. I would suggest the following 
```suggestion
nodejs/**/package-lock.json
nodejs/**/node_modules
```

Similarly for all the other languages (folder and filenames must be according to their respective package managers)

---

Right, makes sense.